### PR TITLE
Add 324 release notes

### PR DESCRIPTION
**Pull request by martint**
Closes https://github.com/prestosql/presto/issues/1862

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-324.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+Release 324
+===========
+
+General Changes
+---------------
+
+* Fix query failure when ``CASE`` operands have different types. (:issue:`1825`)
+* Add support for ``ESCAPE`` clause in ``SHOW CATALOGS LIKE ...``. (:issue:`1691`)
+* Allow references to tables in the enclosing query when using ``.*``. (:issue:`1867`)
+* Configuration properties for iterative optimizer no longer have ``experimental.`` prefix. (:issue:`1875`)
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-324.rst]: Let's merge all of these

File: presto-docs/src/main/sphinx/release/release-324.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+* Configuration properties for cost-based optimizer no longer have ``experimental.`` prefix. (:issue:`1875`)
+* Configuration properties for spill no longer have ``experimental.`` prefix. (:issue:`1875`)
+
+Security Changes
+----------------
+
+* Adding access control check when accessing ``SHOW CREATE TABLE`` and
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-324.rst]: This needs a better explanation (and the grammar is wrong)

File: presto-docs/src/main/sphinx/release/release-324.rst [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  ``SHOW CREATE VIEW``. (:issue:`1517`)
+
+Hive Changes
+------------
+
+* Do not fail ``SHOW GRANTS`` on Hive view when ``hive.security=sql-standard``
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-324.rst]:
```
Allow using ``SHOW GRANTS`` on a Hive view when using the ``sql-standard`` security mode.

File: presto-docs/src/main/sphinx/release/release-324.rst (hunk 2 above)
Comment (martint) [reply presto-docs/src/main/sphinx/release/release-324.rst]: Any suggestions? What about something like "Perform access control checks when displaying table or view definitions with ``SHOW CREATE``"

**Pull request merged**